testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1563,3 +1563,16 @@
   raise 'File injection failed' unless return_code.zero?
   node.run('salt-call --local --file-root=/usr/share/susemanager/salt --module-dirs=/usr/share/susemanager/salt/ --log-level=info --retcode-passthrough state.apply ' + state)
 end
+
+When(/^I copy autoinstall mocked files on server$/) do
+  target_dirs = "/autoinstall/Fedora_12_i386/images/pxeboot/,/autoinstall/SLES15-SP2-x86_64/DVD1/boot/x86_64/loader/,/autoinstall/mock"
+  $server.run("mkdir -p {#{target_dirs}}")
```

Comment:
one pair of curly braces too many

Reply:
I had the same impression Eric, but those outside the variable, are brackets to define a list in bash. I would do it different, but it works, so it works for me.